conanfile.txt

```diff
@@ -0,0 +1,43 @@
+[requires]
+cryptopp/8.2.0@bincrafters/stable
+boost/1.64.0@conan/stable
+gtest/1.8.1@bincrafters/stable
+log4cplus/1.2.1@bincrafters/stable
+OpenSSL/1.1.1@conan/stable
+
+[generators]
+cmake
+
+[options]
+OpenSSL:shared=True
+boost:without_atomic=True
```

Comment:
probably easier to tell 'with' if this option exists

Reply:
I agree it looks a bit ugly, but unfortunately Conan boost options declare 'without' flags.